Gateway/Request/ShopperInteractionDataBuilder.php

```diff
@@ -27,19 +29,25 @@ class ShopperInteractionDataBuilder implements BuilderInterface
     const SHOPPER_INTERACTION_ECOMMERCE = 'Ecommerce';
 
     /**
-     * @var \Magento\Framework\App\State
+     * @var State
      */
     private $appState;
 
+    /** @var PaymentMethods */
+    private $paymentMethodsHelper;
+
     /**
      * RecurringDataBuilder constructor.
      *
      * @param Context $context
+     * @param PaymentMethods $paymentMethodsHelper
      */
     public function __construct(
-        Context $context
+        Context $context,
+        PaymentMethods $paymentMethodsHelper
     ) {
         $this->appState = $context->getAppState();
+        $this->paymentMethodsHelper = $paymentMethodsHelper;
```

Comment:
Not being used right?

Reply:
Yes, updated in https://github.com/Adyen/adyen-magento2/pull/1729/commits/78db43df9283ce1c59d91ca72c072ddd741bf972